Fix #1095: Streamline main README flows; harmonize local <> remote versions

trentmc opened the pull request:
Fixes #1095. This is the big issue with the main work.

This also handles other smaller issues:
- finalizes [#1159](https://github.com/oceanprotocol/ocean.py/issues/1159) -> `set` for envvars in Windows
- fixes [#1202](https://github.com/oceanprotocol/ocean.py/issues/1202) -> how to retrieve datatokens later

calina-c reviewed (dismissed):
I added a small change. I would also like to ask, since the readmes are much changed, if the following are still needed:
- adding instructions for Windows env vars (the last step of #1159) -> this refers to specifying that `export var=value` should be replaced with `set var=value` in Windows command lines
- #1202 -> instructions to retrieving datatokens some time after their creation, based on their address.

For these two, I created #1221 but I will close it for non-interference purposes. I will not delete the branch yet, if you need inspiration.

trentmc commented on READMEs/install.md:
```diff
@@ -53,40 +30,24 @@ pip3 install wheel
-#### Potential issues & workarounds
+## Potential issues & workarounds
 
 - Issue: if you have an Apple M1 processor, `coincurve` and `cryptography` installation may fail due to dependency/compilation issues.
```
Done.

trentmc commented:
> I added a small change. I would also like to ask, since the readmes are much changed, if the following are still needed:
> 
> * adding instructions for Windows env vars (the last step of #1159) -> this refers to specifying that `export var=value` should be replaced with `set var=value` in Windows command lines
> * #1202 -> instructions to retrieving datatokens some time after their creation, based on their address.
> 
> For these two, I created #1221 but I will close it for non-interference purposes. I will not delete the branch yet, if you need inspiration.

Yes, they're needed.

I added them.
